src/CalcManager/ExpressionCommand.cpp

```diff
@@ -256,38 +256,30 @@ const wstring& COpndCommand::GetToken(wchar_t decimalSymbol)
     }
 
     // Remove zeros
-    bool fDigitsFound = false;
-    int trimIdx = 0;
     for (unsigned int i = 0; i < m_token.size(); i++)
     {
         if (m_token.at(i) != chZero)
         {
-            if (m_token.at(i) == decimalSymbol)
+            if (m_token.at(i) == decimalSymbol) 
```

Comment:
nit: you added a trailing space

```suggestion
            if (m_token.at(i) == decimalSymbol)
```